src/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -97,7 +98,7 @@ public static void X509Cert2Test()
             }
         }
 
-        [Fact]
+        [ConditionalFact]
```

Comment:
Seems like this should still just be `[Fact]`

Reply:
This is new trick allowing to throw SkipTestException to do dynamic skip. It is part of your ConditionalFact extension. With Fact from XUnit, throwing will make test fail.